LICENSE.md

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2017 EPFL
```

Comment:
We don't need to use BSD + (c) EPFL, Scalameta is special since it was developed by Phd funds. We can use Apache 2 + copyright Olafur Pall Geirsson and Jonas Fonseca